CMakeLists.txt

```diff
@@ -90,25 +102,30 @@ endif()
 set( CMAKE_EXPORT_COMPILE_COMMANDS ON )
 
 
+#Compile and link options (Should be in experiment project but copied in case CORE is built alone) 
+add_compile_options(-g -O3 -Wextra)
```

Comment:
again, not for now but we should be a little bit careful with just adding compile options. These get added to everything. We should really be using target_compile_options which then only add them for a particular target.